src/Akeneo/Platform/Job/back/ServiceApi/JobInstance/FindJobInstance.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Platform\Job\ServiceApi\JobInstance;
```

Comment:
I believe we should put the implementation in the Infrastructure folder, usually we also name it according to what implem it is, like `SqlFindJobInstance` but I'm not sure what is the rule for new `ServiceApi`s? What do you think @team?
```suggestion
namespace Akeneo\Platform\Job\Infrastructure\Query\JobInstance;
```

Reply:
Agree with you. ServiceAPI only exposes contracts, implementation details should be hide in Infrastructure 👍🏻
I think this is designed like that in others Service API